### PR TITLE
fixing by remove/change the word simply and easily in many documentation

### PR DESCRIPTION
**Pull request by dennisgon**
## Description

I Removed or change the word "just" and "simply" from the sentence following for issue #18702 to appear more interactive and formal.

## Related Issues
Addresses  #18702

### CODE REVIEW

File: docs/docs/answering-it-security.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ At larger companies, such as the Fortune 500, there are Security teams that audi
 If security engineers are interested in your project, some talking points that can help answer their questions include:
 
 - Because Gatsby compiles your site to flat files, rather than having running app servers and databases users are targeting, it reduces the attack surface of the site to outsiders.
-- Gatsby adds a layer of indirection which obscures your CMS -- so even if your CMS _is_ vulnerable, bad actors have no idea where to find it. This is in contrast to systems where bad actors can easily locate the admin dashboard at, e.g., `/wp-admin` and attempt to hack in.
+- Gatsby adds a layer of indirection which obscures your CMS -- so even if your CMS _is_ vulnerable, bad actors have no idea where to find it. This is in contrast to systems where bad actors can be locate the admin dashboard at, e.g., `/wp-admin` and attempt to hack in.
```
Comment (pvdz) [inline docs/docs/answering-it-security.md:10]: I think this is not an example of the issue outlined in the linked issue.

Additionally, I'm certain the change is incorrect.

File: docs/docs/css-in-js.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 title: Enhancing Styles with CSS-in-JS
 ---
 
-CSS-in-JS refers to an approach where styles are written in JavaScript instead of in external CSS files to easily scope styles in components, eliminate dead code, encourage faster performance and dynamic styling, and more.
+CSS-in-JS refers to an approach where styles are written in JavaScript instead of in external CSS files to scope styles in components, eliminate dead code, encourage faster performance and dynamic styling, and more.
```
Comment (pvdz) [inline docs/docs/css-in-js.md:5]: I would rephrase this, "to make it easier to scope styles in components", for example.

File: docs/docs/debugging.md [hunk 3]
```diff
@@ -2,6 +2,6 @@
 title: Debugging
 ---
 
-While building your site, you’ll sometimes encounter bugs. These guides help you set up debugging in Gatsby so you can spot and squash bugs more easily.
+While building your site, you’ll sometimes encounter bugs. These guides help you set up debugging in Gatsby so you can spot and squash bugs more.
```
Comment (pvdz) [inline docs/docs/debugging.md:5]: While the occurrence may be valid (not completely sure), removing the word like this leaves an awkward sentence at best, if not actually an incorrect sentence. Something like "so it makes it easier for you to spot bugs", or something in that line.

File: docs/docs/deploying-to-netlify.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ title: Deploying to Netlify
 This guide walks through how to deploy and host your next Gatsby site on [Netlify](https://www.netlify.com/).
 
 Netlify is an excellent option for deploying Gatsby sites. Netlify is a unified
-platform that automates your code to create performant, easily maintainable
+platform that automates your code to create performant, More easier maintainable
```
Comment (pvdz) [inline docs/docs/deploying-to-netlify.md:8]: Capitalization error. I think in this case you can omit the word without further replacement.

File: docs/docs/gatsby-for-companies.md [hunk 5]
```diff
@@ -16,7 +16,7 @@ Moreover, in software companies, technology decisions are often made bottoms-up
 
 In these environments, if one developer is excited about Gatsby, they can build company projects in Gatsby. If several developers become excited by Gatsby, they may get a team or a group of teams using Gatsby.
 
-If you work in this organization, and want your organization to use Gatsby more, you can go a long way simply by [getting other developers excited by Gatsby](/docs/winning-over-developers/).
+If you work in this organization, and want your organization to use Gatsby more, you can go by [getting other developers excited by Gatsby](/docs/winning-over-developers/).
```
Comment (pvdz) [inline docs/docs/gatsby-for-companies.md:19]: The resulting phrase is awkward. I don't think this is correct.

**Review by pvdz — CHANGES_REQUESTED** [review]
Okay I'm stopping at five suggestions.

Thanks for your PR but this needs more work. Please don't just eliminate the words but consider what the end result looks like.

I'm not a native English speaker and also won't be the final person reviewing this PR, but it is clear to me that this PR cannot be merged as is. Please go through your changes more closely and consider whether or not they in fact meet the requirement of the original issue, and if so, how to properly rewrite them. Sometimes it is sufficient to just drop the word but often it will require rephrasing the sentence to make it "right".

Please don't make us spend more time on this than it would take to do the change ourselves.

Thank you.

**Comment by dennisgon** [comment]
Ok I will fix it

**Review by pvdz — CHANGES_REQUESTED** [review]
Hey, thanks but I don't think this is enough.

Can you please do a `grep -r 'easily' docs/` and `grep -r 'just' docs/` and resolve the issue entirely? Please take care in considering whether or not an occurrence of `easily` or `just` is in fact covered by the issue, or maybe rather a different meaning (like the temporal case outlined in my comment below).

I'd like to see a PR that closes the referenced issue so we can close it. At least for the occurrences of `easily` and `just`. Otherwise we will have to spend even more time going through hundreds of individual PRs and that will take us more time than doing it ourselves.

Thank you :purple_heart:

**Comment by dennisgon** [comment]
Thanks,
I find another word that i already fix it

**Comment by pvdz** [comment]
@dennisgon I'm going to be brutally honest here; I don't think we're going to merge this PR. Just glancing through it I already see a bunch of grammatical problems and you even left a merge conflict in there. As such I'm going to close it.

My guess is that you're participating in Hacktober. That's great! Is there any chance you could find a code change you'd like to submit, rather than a docs change?

**Pull request closed without merging**